docs/technical-reference/code-contributions.md

```diff
@@ -10,7 +10,7 @@ This describes the overall steps to your first code contribution in OpenRefine.
 
 First, why do you want to contribute to OpenRefine?
 - Are you an OpenRefine user who wants to fix or improve an aspect of the tool? Great! Before you dive in and make your changes, make sure you discuss what you want to change with the community first. Check if there is a [GitHub issue](https://github.com/OpenRefine/OpenRefine/issues) on this topic already. If there is one, is there consensus around it? If not, it would be worth opening one. You are also welcome to discuss your plans in the [developer forum](https://forum.openrefine.org/c/dev/8).
-- Do you want to contribute to an open source project and picked OpenRefine for that? Great too! It is useful if you first get a sense of what the tool does. [Install OpenRefine](docs/manual/installing.md) and learn to use it by following [some tutorials](/external_resources) or watching [some videos](https://www.youtube.com/watch?v=B70J_H_zAWM&list=PLYMbXPIhZRlVgxgljjZu6rsFl1yeXXLvp). Once you have a basic understanding of the use cases around OpenRefine, browse through the [list of issues](https://github.com/OpenRefine/OpenRefine/issues) to find an one that you find interesting. You should pick one where you understand what the problem is as a user, you can see why fixing it would be an improvement to the tool. It is also a good idea to pick an issue that matches your technical skills: some require work on the backend (in Java) or in the frontend (Javascript), often both. We try to maintain a list of [good first issues](https://github.com/OpenRefine/OpenRefine/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) which should be easier than others and should not require any difficult design decision.
+- Do you want to contribute to an open source project and picked OpenRefine for that? Great too! It is useful if you first get a sense of what the tool does. [Install OpenRefine](docs/manual/installing.md) and learn to use it by following [some tutorials](/external_resources) or watching [some videos](https://www.youtube.com/watch?v=B70J_H_zAWM&list=PLYMbXPIhZRlVgxgljjZu6rsFl1yeXXLvp). Once you have a basic understanding of the use cases around OpenRefine, browse the [list of issues](https://github.com/OpenRefine/OpenRefine/issues) to find an one that you find interesting. You should pick one where you understand what the problem is as a user, you can see why fixing it would be an improvement to the tool. It is also a good idea to pick an issue that matches your technical skills: some require work on the backend (in Java) or in the frontend (Javascript), often both. We try to maintain a list of [good first issues](https://github.com/OpenRefine/OpenRefine/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) which should be easier than others and should not require any difficult design decision.
```

Comment:
```suggestion
- Do you want to contribute to an open source project and picked OpenRefine for that? Great too! It is useful if you first get a sense of what the tool does. [Install OpenRefine](docs/manual/installing.md) and learn to use it by following [some tutorials](/external_resources) or watching [some videos](https://www.youtube.com/watch?v=B70J_H_zAWM&list=PLYMbXPIhZRlVgxgljjZu6rsFl1yeXXLvp). Once you have a basic understanding of the use cases around OpenRefine, browse the [list of issues](https://github.com/OpenRefine/OpenRefine/issues) to find an one that you find interesting. You should pick one where you understand what the problem is as a user, you can see why fixing it would be an improvement to the tool. It is also a good idea to pick an issue that matches your technical skills: some require work on the backend (in Java) or on the frontend (Javascript), or sometimes both. We try to maintain a list of [good first issues](https://github.com/OpenRefine/OpenRefine/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) which should be easier than others and should not require any difficult design decision.
```

I understand what this introduction is intended for, but it's kind of a lot of text which makes it difficult for a new user to understand. I don't have time to do extensive copy editing right now, so I limited the suggestion to making it agree with the later text.